CodeGeneration/Sources/generate-swiftsyntax/LayoutNode+Extensions.swift

```diff
@@ -84,13 +84,13 @@ extension LayoutNode {
     }
 
     let formattedParams = """
-      - Parameters:
-        - leadingTrivia: Trivia to be prepended to the leading trivia of the node’s first token. \
-      If the node is empty, there is no token to attach the trivia to and the parameter is ignored.
-      \(children.compactMap(generateParamDocComment).joined(separator: "\n"))
-        - trailingTrivia: Trivia to be appended to the trailing trivia of the node’s last token. \
-      If the node is empty, there is no token to attach the trivia to and the parameter is ignored.
-      """.removingEmptyLines
+    - Parameters:
+      - leadingTrivia: Trivia to be prepended to the leading trivia of the node’s first token. \
+    If the node is empty, there is no token to attach the trivia to and the parameter is ignored.
+    \(children.compactMap(generateParamDocComment).joined(separator: "\n"))
+      - trailingTrivia: Trivia to be appended to the trailing trivia of the node’s last token. \
+    If the node is empty, there is no token to attach the trivia to and the parameter is ignored.
+    """.removingEmptyLines
```

Comment:
I suspect this is because you have a `swift-format` installed on your system that wasn’t built from `main`. I’m working on fixing that in principle in https://github.com/apple/swift-syntax/pull/2022 but for now can you just revert the changes to these lines?

Reply:
oh. yeah sure thanks!